client/src/app/+my-library/my-video-channel-syncs/my-video-channel-syncs.component.ts

```diff
@@ -44,6 +44,13 @@ export class MyVideoChannelSyncsComponent extends RestTable implements OnInit {
     this.initialize()
 
     this.videoChannelSyncActions = [
+      [
+        {
+          label: $localize`List imports`,
```

Comment:
Note that the button is not localized anymore.

Is there any way to get the translation back here?

Reply:
It will when I'll update translations I think